esp/scm/ws_resources.ecm

```diff
@@ -88,8 +96,26 @@ ESPresponse [nil_remove, exceptions_inline] WebLinksQueryResponse
     ESParray<ESPstruct ConfiguredWebLink> ConfiguredWebLinks;
 };
 
-ESPservice [auth_feature("ResourceQueryAccess:ACCESS"), version("1.02"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsResources
+ESPStruct [nil_remove] HPCCQueue
+{
+    string Name;
+    ESPenum HPCCQueueType Type;
+    bool QueriesOnly;
```

Comment:
@wangkx 
@ghalliday interested in your take on this.  I know this is the way the configmap currently exposes this internally, but since this interface is exposed externally this will be here forever and is a form of self documentation..

Anyway, just like in the configmap, this treats a query based roxie as a special type of queue.  (As opposed to a truly queue based roxie configured to run workunits).

I realize we may want the convenience of calling 1 function for both, but perhaps the method name should be different and query roxies should be separated out as "roxie services" rather than queues.